src/main/java/net/minestom/server/command/builder/arguments/minecraft/ArgumentItemStack.java

```diff
@@ -36,26 +38,42 @@ public class ArgumentItemStack extends Argument<ItemStack> {
     public static final int INVALID_MATERIAL = 3;
     public static final int INVALID_COMPONENT = 4;
 
+    private final TagStringIO tagStringIO;
+
     public ArgumentItemStack(String id) {
         super(id, true);
+        this.tagStringIO = MinestomAdventure.tagStringIO();
```

Comment:
Is there any reason not to just use the global here? I dont think _not_ wanting heterogeneous lists is a super relevant use case

Reply:
Yeah lets follow the client